dlpy/speech_utils.py

```diff
@@ -38,14 +41,15 @@ def check_framerate(params, framerate):
     Returns
     -------
     boolean
+        Whether the input audio has the desired framerate (True) or not (False).
```

Comment:
B should be capital, as it is a last name

Reply:
Change made and pushed again.